src/Repositories/TenantRepository.php

```diff
@@ -20,7 +20,8 @@ class TenantRepository
      */
     public function query(bool $withTrashed = false)
     {
-        $query = Tenant::query();
+        $class = config('saml2.tenantModel');
```

Comment:
The same here. 

Btw, we could also have something like `Saml2::newModelIntance()` to have one resolver function

Reply:
Would that return the `Model::newModelInstance()` or `Model::query()` that it currently uses? Not sure which is preferred in that case.

Like I have this change stashed if you like it:

```php
<?php

namespace Slides\Saml2;

use Slides\Saml2\Models\Tenant;

class Saml2
{
    public static function tenantModel()
    {
        $class = config('saml2.tenantModel', Tenant::class);
        return $class::query();
    }
}
```